src/Operation/DatabaseCommand.php

```diff
@@ -38,7 +38,7 @@ class DatabaseCommand implements Executable
     /** @var string */
     private $databaseName;
 
-    /** @var array|Command|object */
```

Comment:
This was originally added in https://github.com/mongodb/mongo-php-library/commit/4a764feef33ac21549055e84bd41e4b5820244ca#diff-75b1bd6340fef4dfe6e4d23cb317e77f242e2a7f8344ba6c911b80e9fe87f119R40, but there's no code path where `$command` would be anything other than a Command. I suppose that change was made based on the constructor signature. Good catch.